src/python/gem5/components/boards/abstract_board.py

```diff
@@ -133,7 +146,10 @@ def get_processor(self) -> "AbstractProcessor":
         """
         return self.processor
 
-    def get_memory(self) -> "AbstractMemory":
+    # NOTE: Am I sure that this should be a AbstractMemorySystem instead of a m5.objects.AbstractMemory?
```

Comment:
You're correct. It should be AbstractMemorySystem

Reply:
good to know, thanks

Reply:
I'll probably sneak removing that comment into a future commit